game/src/js/scenes/TitleScene.js

```diff
@@ -53,8 +72,10 @@ export default class TitleScene extends Phaser.Scene {
         }
       });
     }
+
+    // this.cameras.main.startFollow(credits_button);
+    
     
-      
     this.startKey = this.input.keyboard.addKey(Phaser.Input.Keyboard.KeyCodes.ENTER);
```

Comment:
I think we can now remove these two keyboard shortcuts.

Reply:
IMO I actually really like that there's a keyboard shortcut to entering the game as a hidden feature for the lazy.  I'd like to keep it.